src/SFA.DAS.EAS.Web/Orchestrators/Mappers/CommitmentMapper.cs

```diff
@@ -67,6 +71,16 @@ public TransferRequestViewModel MapToTransferRequestViewModel(TransferRequest tr
             };
         }
 
+        public List<TransferConnectionViewModel> MapToTransferConnectionsViewModel(List<TransferConnection> transferConnections)
```

Comment:
This could accept an IEnumerable, so as...
* to allow the calling code to supply any IEnumerable they have instead of forcing them to convert to a List if they don't have a List
* give IEnumerable's lazy evaluation a chance to minimising work

It could also return an IEnumerable (best practice is to return the most specific type, but it's not working with a List, it just creates one last thing from the IEnumerable).

Reply:
done